index.bs

```diff
@@ -263,6 +263,34 @@ so that it works across screen sizes,
 and allows users to choose their own font and font size without causing text to overflow.
 </div>
 
+<h3 id="backwards-compat">Support Existing Content</h3>
+
+Existing content often relies upon expected user agent processing and behavior to function as intended.
+Processing requirements should be specified to ensure that user agents will be able to handle most existing content.
+In particular, it should be possible to process existing web content in current user agents and get results that are compatible with the existing expectations of users and authors,
+based on the behavior of existing user agents.
+
+Content relying on existing user agent behavior can take many forms, such as:
+
+- Relying on features that are entirely proprietary.
+- Depending on specific error handling rules.
+- Depending on certain method names not being defined.
+- Depending on a feature from earlier specifications not being implemented as specified.
+
+When considering changes to legacy features, relative to current implementations and author expectations, the following questions should be considered:
+
+* Would removal of this feature or behavior have significant impact on existing content?
+* Does any of the dependent content occur on particularly high volume websites?
+* Is the dependent content genuinely intended for consumption, rather than occurring solely in test cases or examples?
+* Is the dependent content on the public web, rather than found solely on internal sites with a controlled user environment?
+* Does the dependent content currently work as intended in multiple popular user agents, rather than explicitly targeting only one particular user agent?
+
+The benefit of the proposed change should be weighed against the likely cost of breaking content, as measured by these criteria.
+
+<div class="example">
+
+Many sites use broken markup, such as badly nested elements (`<b>a<i>b</b>c</i>`), and authors have expectations based on the existing error handling results of legacy user agents that such broken markup will still work. We need to define processing requirements that remain compatible with the expected handling of such content.
```

Comment:
"legacy" here also seems to have negative meaning (implying "outdated"), whereas I think what's intended here may be something closer to "existing"